src/Apps/ArtQuiz/ArtQuizInterface/ArtQuizGridComponents.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  Clickable,
+  ClickableProps,
+  Flex,
+  FlexProps,
+  Image,
+  ImageProps,
+  ResponsiveBox,
+  ResponsiveBoxProps,
+  Text,
+  TextProps,
+} from "@artsy/palette"
+import styled from "styled-components"
+import { grid, GridProps } from "styled-system"
+
+// TODO: grid-enable our base Box component in palette
+export const GridClickable = styled(Clickable)<ClickableProps & GridProps>`
+  ${grid}
+`
+export const GridImage = styled(Image)<ImageProps & GridProps>`
+  ${grid}
+`
+export const GridText = styled(Text)<TextProps & GridProps>`
+  ${grid}
+`
+export const GridFlex = styled(Flex)<FlexProps & GridProps>`
+  ${grid}
+`
+export const GridResponsiveBox = styled(ResponsiveBox)<
+  ResponsiveBoxProps & GridProps
+>`
+  ${grid}
+`
```

Comment:
Find this confusing/overkill — would rather just use GridColumns for layout. If a custom grid is actually required I don't think we need to mixin grid to everything.

Reply:
Really the layout in the screenshots just looks like it can be achieved with some simple flexbox — though I haven't seen what the Figma comps look like.

Reply:
I know we already talked about this but another thing i was thinking is the only place were GridColumns (or grid layout) would even be necessary is on the top header row. Something like this would match the layout exactly

```
<Grid /> 
<StretchyImageThing>
<Flex center>
  <button>
  <button>
</Flex>
```

Since everything else is just centered, below to top nav buttons.  Applying the grid to each row is unnecessary since things are simply stacked and centered.

Reply:
lol jinx @dzucconi

Reply:
@damassi & I spoke about this at length. Would love to get w/ you when you're back & have a similar convo @dzucconi. Would prefer to do that synchronously.